File: app/src/main/java/com/young/simpledict/netsoundplayer/request/NetSoundResponse.java
package com.young.simpledict.netsoundplayer.request;

import com.young.simpledict.service.event.BaseEvent;

/**
 * Author: taylorcyang
 * Date:   2014-10-29
 * Time:   12:12
 * Life with passion. Code with creativity!
 */
public class NetSoundResponse extends BaseEvent {
    public static final int RESPONSE_PLAYING = 1;
    public static final int RESPONSE_PAUSED = 2;
    public static final int RESPONSE_COMPLETED = 3;
    public static final int RESPONSE_FAILED = -1;

    public int responseState;
}

File: app/src/main/java/com/young/simpledict/netsoundplayer/request/NetSoundResponse.kt
package com.young.simpledict.netsoundplayer.request

import com.young.simpledict.service.event.BaseEvent

/**
 * Author: landerlyang
 * Date:   2014-10-29
 * Time:   12:12
 * Life with passion. Code with creativity!
 */
class NetSoundResponse(eventCode: Any, var responseState: Int) : BaseEvent(eventCode) {

    constructor() : this(BaseEvent.EMPTY_EVENT_CODE, 0)

    companion object {
        const val RESPONSE_PLAYING = 1
        const val RESPONSE_PAUSED = 2
        const val RESPONSE_COMPLETED = 3
        const val RESPONSE_FAILED = -1
    }
}
